doc/Learn/Mvux/Tutorials/HowTo-MvuxProject.md

```diff
@@ -21,13 +21,13 @@ In this tutorial, you'll learn how to set up an Uno Platform project to use MVUX
 
```

Comment:
I think we can maybe update the VS screenshot for the template here or even point to the new setup doc
https://github.com/unoplatform/uno.extensions/blob/dev/erli/extensions-update-singleproject/doc/Learn/Mvux/Tutorials/HowTo-MvuxProject.md#visual-studio